### ISSUE TITLE
Sorting (Leaderboard) Feature Required

### ISSUE THREAD

**Issue by thisisankit27** [calee0219/leetcode-following#2]
A funttionality of sorting users based on Total question solved, or compared to easy/medium,hard questions solved.

Just to implement a leaderboard functionality

**Comment by calee0219** [comment]
Good idea. I'll take add the feature in my free time (not sure when 😅). Also really welcome to contribution! 🥰
